### PR TITLE
implemented invoice 

### PR DESCRIPTION
**Pull request by Albinseban2021**

### CODE REVIEW

File: lib/functions/fetch_invoice_data.dart [hunk 1]
```diff
@@ -3,14 +3,18 @@ import 'package:http/http.dart' as http;
 import 'package:flutter_dotenv/flutter_dotenv.dart';
 
 class ApiService {
-  static Future<List<dynamic>> getInvoiceData(String shipperId) async {
+  static Future<List<dynamic>> getInvoiceData(
+      String shipperId, String from, String to) async {
     final String invoiceApi = dotenv.get('invoiceApiUrl');
-    final response = await http.get(Uri.parse('$invoiceApi?shipperId=$shipperId'));
+    final response = await http.get(Uri.parse(
+        '$invoiceApi?shipperId=$shipperId&fromTimestamp=$from&toTimestamp=$to'));
```
Comment (Anik-Chakraborty) [inline lib/functions/fetch_invoice_data.dart:10]: change variable name to company id

File: lib/functions/fetch_invoice_data.dart [hunk 2]
```diff
@@ -3,14 +3,18 @@ import 'package:http/http.dart' as http;
...
 
     if (response.statusCode == 200) {
       List<dynamic> data = json.decode(response.body);
       return data;
     } else {
+      print('Error Status Code: ${response.statusCode}');
```
Comment (Anik-Chakraborty) [inline lib/functions/fetch_invoice_data.dart:16]: user debugPrint instead

**Review by Anik-Chakraborty — COMMENTED** [review]
Find these error in console : 
Error: setState() called after dispose(): _EmployeeListRolesScreenState#a2e93(lifecycle state: defunct, not mounted)
This error happens if you call setState() on a State object for a widget that no longer appears in the widget tree (e.g., whose parent widget no longer includes the widget in its build). This error can occur when code calls setState() from a timer or an animation callback.
The preferred solution is to cancel the timer or stop listening to the animation in the dispose() callback. Another solution is to check the "mounted" property of this object before calling setState() to ensure the object is still in the tree.
This error might indicate a memory leak if setState() is being called because another object is retaining a reference to this State object after it has been removed from the tree. To avoid memory leaks, consider breaking the reference to this object during dispose().
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 288:49  throw_
packages/flutter/src/widgets/framework.dart 1103:9                            <fn>
packages/flutter/src/widgets/framework.dart 1137:14                           setState
packages/shipper_app/screens/employee_list_with_roles_screen.dart 192:9       <fn>
dart-sdk/lib/async/zone.dart 1661:54                                          runUnary
dart-sdk/lib/async/future_impl.dart 147:18                                    handleValue
dart-sdk/lib/async/future_impl.dart 784:44                                    handleValueCallback
dart-sdk/lib/async/future_impl.dart 813:13                                    _propagateToListeners
dart-sdk/lib/async/future_impl.dart 584:5                                     [_completeWithValue]
dart-sdk/lib/async/future_impl.dart 657:7                                     callback
dart-sdk/lib/async/schedule_microtask.dart 40:11                              _microtaskLoop
dart-sdk/lib/async/schedule_microtask.dart 49:5                               _startMicrotaskLoop
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 177:15           <fn>

**Comment by Albinseban2021** [comment]
It was not my fault; I had not worked on the team screen.

**Review by Anik-Chakraborty — APPROVED** [review]

**Pull request merged**